### ISSUE TITLE
MacOS 12.1 - M1 Pro

### ISSUE THREAD

**Issue by jcsteele** [joelekstrom/fastmate#58]
App crashes with the following when running from the terminal.  Nothing displayed when running from finder (installed in /Applications)

$ ./Fastmate
Killed: 9

I tried running looking in Console for any type of messages, but I can't find anything.  Interestingly, if I build the code without modifications in XCode it seems to run without issues.

Thoughts on how I can help debug this reproducible error?

**Comment by joelekstrom** [comment]
Hi, thanks. That sounds like the OS is killing it for some reason. Hard to tell without a stack trace, but I imagine it's some gatekeeper issue. Could you maybe try temporarily disabling it and opening? The reason it could be fixed when you build locally may be that it uses your own code signing certificate.

I have neither an M1 or macOS12 yet so my ability to test this is limited at the moment, so I appreciate any help!

**Comment by jcsteele** [comment]
Ah!  Sounds like the issue, and I can confirm at this point that the application is allowed to run.  Thanks so much for the assistance, and thanks for creating this app.  I can't wait to try it out.

For others to resolve this issue: I followed the instructions on https://disable-gatekeeper.github.io/ under the heading of "Disabling Gatekeeper for one application only" and that seemed to take care of the issue.

Feel free to reach out if you need assistance in further Catalina troubleshooting/debug/build issues...happy to help.

**Comment by joelekstrom** [comment]
Thanks for that! Actually, would you mind helping me trying out this beta version, re-enabling Gatekeeper before that? I renewed my Apple ID and had it notarised and properly signed. Just want to make sure it works correctly :) thanks!

[Fastmate-1.8.1-beta.zip](https://github.com/joelekstrom/fastmate/files/7966480/Fastmate-1.8.1-beta.zip)

**Comment by jcsteele** [comment]
I uninstalled all versions, and confirmed gatekeeper settings were restored by trying to run the 1.8.0 version without doing gatekeeper modifications.  As expected, the 1.8.0 app crashed as expected.

I then uninstalled 1.8.0 and installed the beta. It works great.  It does prompt with the typical notifications, etc. but does work after the first run.  I confirmed it runs appropriately from within /Applications/ and outside (specifically, $username/Downloads) as well, so location does not seem to matter.

Can I gather any diags or logs for you for verification?
